pyiron_atomistics/sphinx/base.py

```diff
@@ -2151,6 +2153,13 @@ def from_hdf(self, hdf):
             self.old_version = True
             pass
 
+    def clear_output_for_restart(self):
```

Comment:
Can you call the same `clear_output_for_restart()` function in the `__init__()` function of the Output class? This hopefully prevents us from changing both independent from each other.